selfdrive/common/params.cc

```diff
@@ -137,7 +137,7 @@ std::unordered_map<std::string, uint32_t> keys = {
     {"LastSystemShutdown", CLEAR_ON_MANAGER_START},
     {"LastUpdateException", PERSISTENT},
     {"LastUpdateTime", PERSISTENT},
-    {"LiveParameters", PERSISTENT},
+    {"LiveParameters", PERSISTENT | CLEAR_ON_CAR_CHANGED},
```

Comment:
You can update `CalibrationParams` as well.

Reply:
never mind, let's leave calibrationd for what it is. It currently also doesn't reset on car change.